### ISSUE TITLE
Arduino UNO board not available after installation "Arduino AVR boards"

### ISSUE THREAD

**Issue by Erriez** [arduino/arduino-pro-ide#69]
The Arduino UNO board is not available after "Arduino AVR boards" installation on Windows 10 64-bit.

**To Reproduce**
1. Start new Arduino IDE
2. Click Tools | Board manager...
3. Install "Arduino AVR boards by Arduino" (Installation successful)
4. Select other boards & ports | UNO board grayed out. Restart IDE does not help.

**Expected behavior**
Arduino/Genuine board must be available (grayed out) including a serial port (nothing found).

**Screenshots**
![image](https://user-images.githubusercontent.com/8849873/67231522-c105a380-f43f-11e9-8f06-93fc42719d8b.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 64-bit

**Comment by rei-vilo** [comment]
At launch, the IDE displays a cryptic message.

<img width="512" alt="Capture d’écran 2019-10-25 à 18 55 53" src="https://user-images.githubusercontent.com/951776/67589975-17cdee80-f75a-11e9-8387-9552f9c3581f.png">

Might be related.

Arduino Pro IDE Alpha 0.0.2 on macOS 10.15.

**Comment by mastrolinux** [comment]
@rei-vilo no, that is not related and the error is coming from the language-server. Are you still unable to view the serial ports with https://github.com/arduino/arduino-pro-ide/releases/tag/v0.0.2-alpha.preview version? @Erriez

**Comment by rei-vilo** [comment]
The port shows now on with Alpha 0.0.2. I've close #20.

**Comment by Erriez** [comment]
Serial port is now visible in board window. It is possible to upload a sketch, but the serial terminal does not work. The serial port is not opened and the output window remains empty.

![image](https://user-images.githubusercontent.com/8849873/67709026-819a0280-f9bd-11e9-9224-2a33f7219e0d.png)


The sketch below works perfect with the original Arduino IDE:
```
void setup() 
{
    Serial.begin(115200);
    Serial.println(F("Hello world!"));
}

void loop() 
{
    Serial.println(F("."));
    delay(1000);
}
```
![image](https://user-images.githubusercontent.com/8849873/67709167-b017dd80-f9bd-11e9-99f9-87a8cbd68bb8.png)

**Comment by rei-vilo** [comment]
The Arduino Nano 33 BLE does print on the console under macOS when using Alpha 0.0.2.

<img width="970" alt="Capture d’écran 2019-10-28 à 20 07 25" src="https://user-images.githubusercontent.com/951776/67709677-b65a8980-f9be-11e9-9987-02249e06e222.png">

**Comment by Erriez** [comment]
I'm on Windows. The serial port is not opened for some reason.

**Comment by kittaakos** [comment]
There are multiple things logged to one GH issue. Let me clarify them:


> Arduino/Genuine board must be available (grayed out) including a serial port (nothing found).

Now it is.

![image](https://user-images.githubusercontent.com/1405703/78878351-8e358300-7a52-11ea-8ef6-25109a983c0e.png)

----

> At launch, the IDE displays a cryptic message.
> 
> <img alt="Capture d’écran 2019-10-25 à 18 55 53" width="512" src="https://user-images.githubusercontent.com/951776/67589975-17cdee80-f75a-11e9-8387-9552f9c3581f.png">
> 
> Might be related.

No, it's unrelated.

----

> The serial port is not opened and the output window remains empty, no matter which serial baudrate is selected.

The monitor had some issues when the line was not terminated with `\n`. It was fixed via #201.
